tool/stats_lib.dart

```diff
@@ -117,11 +126,31 @@ CompareResult compareResult(
   bool throwOnError = false,
   bool verboseFail = false,
   bool verboseLooseMatch = false,
+  List<String> extensions = const [],
 }) {
   String output;
+  final inlineSyntaxes = <InlineSyntax>[];
+  final blockSyntaxes = <BlockSyntax>[];
+
+  for (final extension in extensions) {
+    switch (extension) {
+      case 'autolink':
+        inlineSyntaxes.add(AutolinkExtensionSyntax());
+        break;
+      case 'strikethrough':
+        inlineSyntaxes.add(StrikethroughSyntax());
+        break;
+      case 'table':
+        blockSyntaxes.add(const TableSyntax());
```

Comment:
probably good to add a `default` case and throw if we hit unexpected input, right?

Reply:
Fixed